src/pages/organisationUnits/form/types.ts

```diff
@@ -0,0 +1,5 @@
+import { OrganisationUnit } from '../../../types/generated'
+
+export type FormValues = Omit<OrganisationUnit, 'parent'> & {
+    parent: { id: string }[]
```

Comment:
I think we should add a small comment as to why we make this an array - when the model is just a reference. (Due to the OrgUnit selector operating on arrays).